manifests/dev/vsphere-7.0u2/vanilla/rbac/vsphere-csi-node-rbac.yaml

```diff
@@ -4,26 +4,26 @@ metadata:
   name: vsphere-csi-node
   namespace: kube-system
 ---
-kind: ClusterRole
```

Comment:
Are you cherry-picking this change to 2.1 as well? Can you also make the same change in https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/dev/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml and https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/dev/vsphere-7.0u2/vanilla/deploy/validatingwebhook.yaml too?

cc: @divyenpatel

Reply:
Not sure if this will be cherry-picked but the changes you have suggested are needed in master.